go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/stolostron/submariner-addon
 go 1.18
 
 require (
+	github.com/Azure/go-autorest/autorest/azure/auth v0.5.7
```

Comment:
Small typo in the commit message. If you happen to respin the PR
preperation -> preparation